src/fluree/db/query/exec/select.cljc

```diff
@@ -60,53 +60,73 @@
       :sparql (with-meta selector {`format-value select.sparql/format-wildcard-selector-value})
       (with-meta selector {`format-value select.fql/format-wildcard-selector-value}))))
 
-(defrecord AggregateSelector [agg-fn]
+(defrecord AggregateSelector [agg-fn streaming-agg]
```

Comment:
I think I would prefer a `StreamingAggregateSelector` record with its own separate `ValueSelector` implementation. Having this opaque map complects multiple behavior traits into the same record.

Reply:
Addressed in 7c57470b9a52c69e9ab2b6f69a0219e4ab4fa64f